todolist/settings.py

```diff
@@ -53,28 +53,15 @@
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 )
 
-ROOT_URLCONF = "todolist.urls"
-
-WSGI_APPLICATION = "todolist.wsgi.application"
-
```

Comment:
Why you've removed those configurations, your server working not in the right way.
<img width="1431" alt="image" src="https://github.com/mate-academy/devops_todolist_docker_core_task_2_volumes/assets/45535903/4556f680-0536-4e0e-8829-05c012db5469">